signingscript/Dockerfile

```diff
@@ -4,17 +4,19 @@ RUN groupadd --gid 10001 app && \
     useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
 
 RUN apt-get update \
- && apt-get install -y zipalign osslsigncode \
+ && apt-get install -y zipalign osslsigncode cmake clang \
  && apt-get clean \
  && ln -s /app/docker.d/healthcheck /bin/healthcheck
 
 COPY . /app
 RUN chown -R app:app /app
 
+RUN cd /app/signingscript/docker.d && bash build_msix_packaging.sh && cp msix-packaging/.vs/bin/makemsix /usr/bin && cp msix-packaging/.vs/lib/libmsix.so /usr/lib && cd .. && rm -rf msix-packaging
+
 USER app
 WORKDIR /app
 
-RUN python -m venv /app \
+RUN cd /app && python -m venv /app \
```

Comment:
Guessing this is because we cd'd into `/app/signingscript` above?

Reply:
I don't recall; I'll see if I can simplify it though.